src/store/messagesStore.js

```diff
@@ -501,6 +501,20 @@ const actions = {
 			})
 		}
 
+		if (message.systemMessage === 'call_started') {
```

Comment:
```suggestion
		if (message.systemMessage === 'call_started' && context.getters.conversation(token).lastActivity < message.timestamp) {
```

We need to check in addition if we're processing a new message now, or an old one, to not trigger a call UI by coincidence
Not sure about the correct line patch, but should be understndable